lib/systems/default.nix

```diff
@@ -183,6 +183,26 @@ let
          || isWasm                                                     # WASM
         ) && !isStatic;
 
+      # Controls whether libunwind is enabled.
+      hasStackUnwinding = !(final.isWasm || final.isFreeBSD || final.isDarwin || final.isNone);
+
+      # You often don't want excepetions in an embedded setting, disable them by default.
+      hasExceptions = !(final.isNone || final.isWasm);
+
+      # If there is no kernel (`isNone`) we assume no threads. WASM can
+      # supposedly support threads in certain cases, so this might change in
+      # the future.
+      hasThreads = !(final.isNone || final.isWasm);
+
+      hasFilesystem = !(final.isNone || final.isWasm);
```

Comment:
I think this is the only one without an example usecase.

Reply:
Added a comment with some explanation.

Reply:
I'm not sure about this one either.  It sounds like on wasm you _sometimes_ have a filesystem, so this isn't really a property of the platform.